core/controllers/topic_editor.py

```diff
@@ -166,21 +161,6 @@ class EditableSubtopicPageDataHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
-    def _require_valid_version(
```

Comment:
Why are we deleting this?

Reply:
This function is not being called in this controller. This function is also defined in EditableTopicDataHandler where it does get called

Reply:
OK, thank you for clarifying.